### ISSUE TITLE
In telegram, can't connect wallet

### ISSUE THREAD

**Issue by differs** [kang5647/bnb-telegram-demo#1]
I tried use tokenpocket to test

**Comment by dhadx** [comment]
> I tried use tokenpocket to test

You got connected?

**Comment by kang5647** [comment]
Yes, the current issue with connecting wallet in telegram mini app (on mobile) is that the re-direct won't work. Some workarounds will be to use email auth solution like Particles Network, or Avatar wallet which supports qr copy copy pasting. 

![b32fafac-594d-414f-bee0-b21a15842c67](https://github.com/user-attachments/assets/b0441177-9955-4863-a4f0-bd7ac196c463)
![d43be7b2-3b57-4bfe-9f03-093b1629e6ec](https://github.com/user-attachments/assets/208f529b-3056-4002-8baf-364e93912fc5)
![1d41cb66-4fa4-4762-80a4-971674455021](https://github.com/user-attachments/assets/c5a57fe5-ca76-4ef7-9418-25e04e8cd71b)

**Comment by dhadx** [comment]
> Yes, the current issue with connecting wallet in telegram mini app (on mobile) is that the re-direct won't work. Some workarounds will be to use email auth solution like Particles Network, or Avatar wallet which supports qr copy copy pasting. 
> 
> 
> 
> ![b32fafac-594d-414f-bee0-b21a15842c67](https://github.com/user-attachments/assets/b0441177-9955-4863-a4f0-bd7ac196c463)
> 
> ![d43be7b2-3b57-4bfe-9f03-093b1629e6ec](https://github.com/user-attachments/assets/208f529b-3056-4002-8baf-364e93912fc5)
> 
> ![1d41cb66-4fa4-4762-80a4-971674455021](https://github.com/user-attachments/assets/c5a57fe5-ca76-4ef7-9418-25e04e8cd71b)
> 
> 


Since you got connected already.
